File: LANGUAGES/3) JAVA SCRIPT/4) IF/15.js
let a = 5, b = 7, c = 9
let max1 = a, max2 = b
if (b > a && c > a) {
    max1 = c
}
if (a > b && c > b) {
    max2 = c
}
console.log(max1, max2)

File: LANGUAGES/1) C++/4) IF/15.cpp
#include <iostream>

using namespace std;

int main()
{
	int a, b, c, max1, max2;
	cout << " A = ";
	cin >> a;
	cout << " B = ";
	cin >> b;
	cout << " C = ";
	cin >> c;

	if (a > b and b > c or b > a and a > c)
	{
		max1 = a;
		max2 = b;
	}
	if (b > c and c > a or c > b and b > a)
	{
		max1 = b;
		max2 = c;
	}
	if (c > a and a > b or a > c and c > b)
	{
		max1 = c;
		max2 = a;
	}

	cout << " Max1 : " << max1 << endl;
	cout << " Max2 : " << max2;
	return 0;
}